Danki2/Assets/Scripts/Actor/Mortal/Player/Player.cs

```diff
@@ -9,6 +9,18 @@ public class Player : Mortal
 
     protected override void Act()
     {
-        // TODO: Act according to key-presses.
+        this.Move();
+    }
+
+    private void Move()
+    {
+        var _moveDirection = Vector3.zero;
+
+        if (Input.GetAxis("Horizontal") > 0) _moveDirection.x += 1f;
```

Comment:
For some reason the movement feels a bit weird (like the player keeps moving after you let go of the button). I think it has something to do with "Gravity" value of the input, it's like the value of the axis slow degrades over time when you release it rather than just going off. I don't really know much about Unity input, it seems like using `Input.GetAxisRaw(...)` fixes this, but I don't really understand what's going on